app/javascript/App/bundles/NewsfeedPage/NewsfeedPage.tsx

```diff
@@ -317,6 +318,10 @@ class NewsfeedPage extends React.Component<Props, State> {
 
   public render() {
     if (this.props.isMobile) {
+      const coinClickHandler = (coinData: CoinLinkData) => {
+        this.props.history.push(`/news/${coinData.slug}`)
+        this.setState({ ActiveMobileWindow: 'None' })
```

Comment:
@jessehon Is this necessary for all links? Not sure exactly when/how/where to use this...

Reply:
At the moment, I believe we still need `ActiveMobileWindow` otherwise the wrong pane would end up showing. Ideally this should just be inferred from the router state but for now, we still need to manually update it.

Reply:
Awesome, thanks for the context! 🙏